### PR TITLE
Checkboxes

### PR DESCRIPTION
**Pull request by BryanGK**

### CODE REVIEW

File: packages/form-schema/src/components/CheckboxesWidget.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```
Comment (BryanGK) [inline packages/form-schema/src/components/CheckboxesWidget.js]: This file contained a lot of es-lint errors when I copied it into the project. I'm not entirely sure how/what needs fixing here.

File: packages/form-schema/src/widgetWrapper.tsx [hunk 2]
```diff
@@ -45,6 +45,14 @@ const Wrapper = (Component, inputType: string = '') => {
       formProps.value = true;
     }
 
+    if (inputType === 'checkboxes') {
+      return (
+        <div>
+          <Component />
```
Comment (BryanGK) [inline packages/form-schema/src/widgetWrapper.tsx]: I wasn't sure where to find the props that needed to be passed in here

File: packages/form-schema/src/components/CheckboxesWidget.js (hunk 1 above)
Comment (jlangy) [reply packages/form-schema/src/components/CheckboxesWidget.js]: es errors or ts errors? We haven't been using prop types, just typescript for props s lines 59-82 can go. If they are ts-errors I would put `// ts-ignore` at the top of the file first just to check it's working

File: packages/form-schema/src/components/CheckboxesWidget.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  const { enumOptions, enumDisabled, inline } = options;
+  return (
+    <div className="checkboxes" id={id}>
+      {enumOptions.map((option, index) => {
+        const checked = value.indexOf(option.value) !== -1;
+        const itemDisabled = enumDisabled && enumDisabled.indexOf(option.value) !== -1;
+        const disabledCls = disabled || itemDisabled || readonly ? 'disabled' : '';
+        const checkbox = (
+          <span>
+            <input
```
Comment (jlangy) [inline packages/form-schema/src/components/CheckboxesWidget.js]: For this input, you will want to import the bcgov Checkbox component and use that instead, removing type="checkbox"

File: packages/form-schema/src/widgetWrapper.tsx (hunk 2 above)
Comment (jlangy) [reply packages/form-schema/src/widgetWrapper.tsx]: I think passing in <Component {...formProps} /> should mostly work, there may be some to add/remove

File: packages/form-schema/src/components/CheckboxesWidget.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/no-autofocus */
```
Comment (BryanGK) [inline packages/form-schema/src/components/CheckboxesWidget.js]: I added these lines to ignore the errors I was getting

File: packages/form-schema/src/components/CheckboxesWidget.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  const { enumOptions, enumDisabled, inline } = options;
+  return (
+    <div className="checkboxes" id={id}>
+      {enumOptions.map((option, index) => {
+        const checked = value.indexOf(option.value) !== -1;
+        const itemDisabled = enumDisabled && enumDisabled.indexOf(option.value) !== -1;
+        const disabledCls = disabled || itemDisabled || readonly ? 'disabled' : '';
+        const checkbox = (
+          <span>
+            <Checkbox
```
Comment (BryanGK) [inline packages/form-schema/src/components/CheckboxesWidget.js]: You meant like this?

**Comment by BryanGK** [comment]
In `widgets.tsx` I'm now getting errors at the top stating it can no longer find Checkbox, RadioButton, or Input modules. 
Screenshot from 2022-01-05 13-56-32

And the big error I'm getting is that stating it cannot find the `CheckboxesWidget` module, am I not export/importing that properly?
Screenshot from 2022-01-05 13-53-01

**Review by jlangy — APPROVED** [review]

**Pull request merged**